Symfony/CS/Tests/Fixer/PSR2/EofEndingFixerTest.php

```diff
@@ -52,17 +52,46 @@ public function provideCases()
 ',
             ),
             array(
-                "<?php\r\$a = 1;\n",
-                "<?php\r\$a = 1;",
+                "<?php\r\$a = 4;\n",
+                "<?php\r\$a = 4;",
             ),
             array(
-                "<?php\r\$a = 1;\n",
-                "<?php\r\$a = 1;\r",
+                "<?php\r\$a = 5;\n",
+                "<?php\r\$a = 5;\r",
```

Comment:
`CR` may exist as line ending on mac ecosystem.
And this fixer should not convert one end of line into another.
If there is any new line at the end (`CR`, `CR`+`LF` or `LF`) it must not be changed into `LF`. It is a work for a separate fixer (and we already have that one)